### PR TITLE
Metrics Support for demo

### PR DESCRIPTION
**Pull request by Sumukh-Phalgaonkar**
Added metrics setup to the demo. Now with  same `docker compose up` as before, we will spin up docker containers for prometheus and grafana. We also have sample dashboards grafana-dashboard directory which we can directly import.

### CODE REVIEW

File: cdc-quickstart-kafka-connect/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ - job_name: 'kafka-connect-host'
+   static_configs:
+   - targets: ['kafka-connect:9876']
+
+ - job_name: 'yugabyted'
+   metrics_path: '/prometheus-metrics'
+   static_configs:
+   - targets: ['172.165.20.83:9000']
+
+# Add more jobs here to get metrics from multiple nodes
```
Comment (fourpointfour) [inline cdc-quickstart-kafka-connect/prometheus.yml]: Need one extra line at the end of the file.

**Comment by vrajat** [comment]
Lets move the metric setup in the docker file to yugabyte/debezium-connector-yugabytedb

**Comment by vrajat** [comment]
@Sumukh-Phalgaonkar Can you open a PR to move the metrics/jmx to debezium connector repo?

File: cdc-quickstart-kafka-connect/prometheus.yml (hunk 1 above)
Comment (Sumukh-Phalgaonkar) [reply cdc-quickstart-kafka-connect/prometheus.yml]: Added

File: cdc-quickstart-kafka-connect/prometheus.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+scrape_configs:
+
+ - job_name: 'kafka-connect-host'
+   static_configs:
+   - targets: ['kafka-connect:9876']
+
+ - job_name: 'yugabyted'
+   metrics_path: '/prometheus-metrics'
+   static_configs:
+   - targets: ['172.165.20.83:9000']
```
Comment (vrajat) [inline cdc-quickstart-kafka-connect/prometheus.yml]: Is this IP always the same?

**Review by vrajat — COMMENTED** [review]
Changes look good in general. I have one question on an hard coded IP

**Comment by vrajat** [comment]
@vaibhav-yb for the quick start should we use confluent or Apache Kafka? I prefer confluent as the rest of the examples use that it IMO is more user friendly.

File: cdc-quickstart-kafka-connect/prometheus.yml (hunk 2 above)
Comment (Sumukh-Phalgaonkar) [reply cdc-quickstart-kafka-connect/prometheus.yml]: No it was my dev server's IP. I have removed hard coded part now

**Review by vrajat — COMMENTED** [review]
One more thing. Delete the monitoring directory?

**Comment by Sumukh-Phalgaonkar** [comment]
> One more thing. Delete the monitoring directory?

We can delete it. Should I delete it as a part of this PR or should I make another PR for this?

**Comment by vrajat** [comment]
You can delete it as part of this PR

**Review by vrajat — APPROVED** [review]

**Pull request merged**